### PR TITLE
Build: Update Travis-CI config

### PR DESCRIPTION
**Pull request by leobalter**
Ref: http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade#How-can-I-use-container-based-infrastructure%3F"

### CODE REVIEW

**Comment by leobalter** [comment]
It runs faster, but it doesn't prevent browserstack from failing.

**Comment by jzaefferer** [comment]
While we're here, can we also cache node_modules?

```yml
cache:
  directories:
  - node_modules
```

**Comment by leobalter** [comment]
done, squashed to a single commit

**Comment by jzaefferer** [comment]
Looks good!

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,8 @@ before_install:
   - true && `base64 --decode <<< ZXhwb3J0IEJST1dTRVJTVEFDS19LRVk9SllzeHJrVWk5aGJGVndkdW44ZUsK=`
 script:
   - npm run-script ci
+sudo: false
+cache:
+  directories:
+    - node_modules
+
```
Comment (arthurvr) [inline .travis.yml]: remove the blank line?

**Comment by arthurvr** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (leobalter) [inline .travis.yml]: thanks!

**Pull request merged**